docs/import/create_disaster_lib.js

```diff
@@ -114,34 +116,72 @@ const stateBasedDataTemplate = Object.freeze({
 const stateAssetDataTemplate = Object.freeze(
     {...commonAssetDataTemplate, stateBasedData: stateBasedDataTemplate});
 
+/**
+ * Where building count comes from.
+ * @type {Object}
+ * @param {number} BUILDING Indicates that there is a separate buildings asset.
+ *     Asset can contain geometries (in which case each geometry corresponds to
+ *     a building), or just have rows, in which case each row has a building
+ *     count that will be joined to the poverty asset. `buildingGeoid` will
+ *     be the joining column and `buildingKey` the column with the count.
+ * @param {number} POVERTY Indicates that poverty asset already has a building
+ *     count per district. `buildingKey` will identify the column.
+ * @param {number} DAMAGE Indicates that damage asset contains geometries for
+ *     all buildings, not just damaged ones, so the building count can be taken
+ *     from it. If specified, `flexibleData` fields `noDamageKey` and
+ *     `noDamageValue` must be set if `assetData.damageAssetPath` is. (If this
```

Comment:
Maybe I'll see this in code below, so DAMAGE is the default setting for if there's no damage asset? Maybe just create a 4th setting for clarity?